js-api-doc/legacy/options.d.ts

```diff
@@ -508,6 +508,13 @@ export interface LegacySharedOptions<sync extends 'sync' | 'async'> {
    * @compatibility dart: "1.43.0", node: false
    */
   logger?: Logger;
+
+  /**
+   * If this option is set to `'node'`, Sass will use the built-in Node Package
+   * Importer to resolve Sass files with a `pkg:` URL scheme.
+   */
```

Comment:
Similar comments to `nodePackageImporter`. This doesn't need to be as detailed, but it should link to `nodePackageImporter` for full information.

Reply:
Updated in 9d1d3d2c4d9f04f9a3d4021cf06ee93fe089e531